config/default.ts

```diff
@@ -28,12 +28,22 @@ export default {
   auth: {
     accessSecret: process.env.JWT_ACCESS_TOKEN_SECRET || 'secret',
     refreshSecret: process.env.JWT_REFRESH_TOKEN_SECRET || 'secret',
+    resetSecret: process.env.JWT_RESET_TOKEN_SECRET || 'secret',
     accessExpires: Number.parseInt(process.env.JWT_ACCESS_TOKEN_EXPIRES as string, 10) || '1h',
     refreshExpires: process.env.JWT_REFRESH_TOKEN_EXPIRES || '30d',
+    resetExpires: process.env.JWT_RESET_TOKEN_EXPIRES || '30d',
   },
   paging: {
     pageNumberDefault: 0,
     pageSizeDefault: 5,
     sortingDirectionDefault: 'ASC',
   },
+  mailService: {
+    service: process.env.MAIL_SERVICE_SERVICE || 'Dummy',
+    senderMail: process.env.MAIL_SERVICE_SENDER || 'resetpassword@ptushki.by',
+    subject: {
+      passwordChange: 'Your password has been changed',
+      passwordReset: 'Your password has been reset',
```

Comment:
It doesn't seem consistent. The body of the email is stored in the service and the headers in the config, what if we need to add localization?

Reply:
I moved subjects for email from config file to templates. Now it is more consistent. But I don't think what we need to add localization right now for mvp.

Reply:
I agree, it was a matter of perspective.